api/v1beta1/kafkacluster_types.go

```diff
@@ -736,6 +741,23 @@ func (eConfig *EnvoyConfig) GetResources() *corev1.ResourceRequirements {
 	}
 }
 
+// GetConcurrency returns envoy concurrency.
+// Defines the number of worker threads envoy pod should run.
+// If not specified defaults to the number of hardware threads on the underlying kubernetes node.
+func (eConfig *EnvoyConfig) GetConcurrency() int {
+	if eConfig.AdditionalConfig == nil {
+		return 0
+	}
+	if value, ok := eConfig.AdditionalConfig["concurrency"]; ok {
```

Comment:
Let's make `concurrency` a const and add a link to the documentation in the comment.